### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by crashzk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+# ServerAdvertisement3
+Very useful plugin for your server with which you can print messages to players. Messages are stored in file.
+
+## Features
+- Message Types:
+	- Chat messages ***(Type T)*** - Color Support;
+	- HUD messages ***(Type H)*** - R G B Color;
+	- Center Text messages ***(Type C)*** - Support Only HTML **`hex`** colors;
+- Multi-language support; ***(Use [Alpha-2 Codes](https://en.wikipedia.org/wiki/ISO_3166-1))***
+- Up to 9 lines per message; **(`Use \n` to break lines)**
```
Comment (Adrianilloo) [inline README.md]: Per #16 this is not fixed anymore. Lines amount can vary and are implicitly limited by the enough large 2K buffer (i.e. max. text size).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
+# ServerAdvertisement3
+Very useful plugin for your server with which you can print messages to players. Messages are stored in file.
+
+## Features
+- Message Types:
```
Comment (Adrianilloo) [inline README.md:5]: Also there is my newer Top Menu type per #16 (which just permits to adjust the `color` setting in the CFG):

https://user-images.githubusercontent.com/15228896/164123861-13dcb895-d5ae-4a77-b022-bf41b12f485c.jpg

File: README.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+{SERVERNAME}
+{PLAYERNAME}
+{ADMINSONLINE}
+{TIMELEFT}
+{STEAMID}
+{CONVAR} -> {mp_friendlyfire}, {sv_cheats}, or convar from any pluigin
+{CURRENTDATE}
+{CURRENTDATE_US}
+{VIPONLINE} - Flag A
+```
```
Comment (Adrianilloo) [inline README.md:59]: Also `{TAG}` per #16

**Comment by crashzk** [comment]
I made the corrections you quoted **@Adrianilloo**. If I believe I need to change something else in this README, I've attached it here too, just edit and attach again and I'll update.

[README.MD](https://github.com/ESK0/ServerAdvertisement3/files/8603190/README.MD)

**Pull request merged**